### DISCUSSION TITLE
continuous deployment to s3 or database?

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-cd#8327] [Q&A]
apart from ensuring k8s objects on git are synced to k8s cluster, can argocd also sync things like scripts/config files/jars to s3 or Sql files against a database?

**Comment by samyak-jn** [comment]
Hi @tooptoop4,

I'm quite not sure about that, But mostly, Argo CD is implemented as a kubernetes controller which continuously monitors running applications and compares the current, live state against the desired target state (as specified in the Git repo). So whatever you push to the git, it maintain it's sync with the target site. I always go back to [architecture design](https://argo-cd.readthedocs.io/en/stable/operator-manual/architecture/) to get more clarity.

**Reply by tooptoop4** [reply]
https://github.com/schemahero/schemahero and gitSync to S3 wf based on git event seems to be what I want
